### PR TITLE
[Config] articles chronology in comment

### PR DESCRIPTION
**Pull request by ghost**
"Loading resources" is next to currnet section "Caching based on Resources", not previews

### CODE REVIEW

File: components/config/caching.rst [hunk 1]
```diff
@@ -38,7 +38,7 @@ should be regenerated::
         $resources = array();
 
         foreach ($yamlUserFiles as $yamlUserFile) {
-            // see the previous article "Loading resources" to
+            // see the next article "Loading resources" to
```
Comment (xabbuh) [inline components/config/caching.rst]: Actually, it isn't the very next article neither (which is "Defining and Processing Configuration Values"). So I think we can better reword the comment by simply removing "previous". What do you think?

Comment (karzz) [inline components/config/caching.rst]: I think it is good idea.

**Review by javiereguiluz — APPROVED** [review]
👍 

Status: reviewed

**Review by HeahDude — APPROVED** [review]

**Comment by xabbuh** [comment]
Good catch, thanks @karzz.

**Pull request closed without merging**